index.html

```diff
@@ -3,15 +3,186 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="mobile.css">
     <title>Jery Mey Capstone Project</title>
 </head>
 <body>
+    <header>
+        <div class="align">
+            <p>.</p>
+        <ul class="social-header">
+            <li><a href=""><img src="images/Rectangle.png" alt=""></a></li>
+            <li><a href=""><img src="images/twitter.png" alt=""></a></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+      </div>
+    </header>
+    <nav class="navbar">
+        <div class="container">
+            <div class="hamb">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>   
+            <div class="logo"><a href="https://jerymey.github.io/Capstone/"><img src="images/Market (1).png" alt=""></a></div>
+            <ul class="nav">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link">about</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Program</a>
+                </li> 
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Join</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Sponser</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">New</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link campaign">CC Campaign</a>
+                </li>
+            </ul>
+        </div>
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **The pages should look almost identical to the original design.** Small adjustments like text or image changes are acceptable. So Kindly Adjust the  the Nav-bar elements have similar spacing, borders, shadows, and paddings than the original design. Kindly Fix the issues 
- Your Nav-bar Design look likes this : 
![nava](https://github.com/jerymey/Capstone/assets/93598189/8cfdc68e-bb12-4381-96bf-815644d0e1d8)

- The Original Design Look likes this : -

![nava 1](https://github.com/jerymey/Capstone/assets/93598189/11b87405-f5bb-4a3f-b661-985f7a1daa93)